packages/lodestar-cli/src/db.ts

```diff
@@ -0,0 +1,34 @@
+import {fromHexString} from "@chainsafe/ssz";
+import {BeaconDb, LevelDbController} from "@chainsafe/lodestar/lib/db";
+import {WinstonLogger, LogLevel} from "@chainsafe/lodestar-utils";
+
+import {getBeaconPaths} from "./cmds/beacon/paths";
+import {mergeConfigOptions} from "./config/beacon";
+import {getBeaconConfig} from "./util";
+
+const rootDir = ".altona";
+const bucket = "badBlock";
+const action = "delete";
+const value = "0xe37b83f1e25ce3ebc2e10fa308413160eb36e4de6d295079104222d67229136c";
+
+(async function() {
```

Comment:
Why does this function need to run in the body of the script?

Reply:
doh, this file isn't meant to be added, its just some `beacon db` script i've been toying with